modules/persistent-storage-csi-tp-enable.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+// persistent-storage-csi-azure.adoc
+
+ifeval::["{context}" == "persistent-storage-csi-vsphere"]
+:vsphere:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-azure"]
+:azure:
+endif::[]
+
+[id="csi-tp-enable_{context}"]
+= Enabling the {FeatureName} CSI driver operator
+To enable the {FeatureName} Container Storage Interface (CSI) driver operator, you must enable feature gates.
```

Comment:
Should we be specific about the feature gate that needs to be enabled?

Reply:
Yup. Done!